app/konnect-platform/support-policy.md

```diff
@@ -2,11 +2,12 @@
 title: Version Support Policy
```

Comment:
Should this be "Version Support for Kong Ingress Controller (Enterprise)"

Reply:
Only line 83 is the Kong Ingress Controller table. Previously it was titled something differently. The doc is our product version support policy doc.